### DISCUSSION TITLE
Do you understand the "Invalid Platform Architecture" of the USMT report?

### DISCUSSION THREAD

**Discussion by Deng-Xian-Sheng** [belowaverage-org/SuperGrate#123] [General]
This is log

```
2024-06-18 08:41:10, Info                  [0x080362] Dumping all migration units: END
2024-06-18 08:41:10, Info                  [0x000000] Completed deserialization of migration engine state
2024-06-18 08:41:10, Info                  [0x000000] Leaving MigOpenIMGStore method
2024-06-18 08:41:10, Info                  [0x000000] Entering MigGetStorePlatform method
2024-06-18 08:41:10, Info                  [0x000000] Leaving MigGetStorePlatform method
2024-06-18 08:41:10, Info                  [0x000000] Entering MigGetRealPlatform method
2024-06-18 08:41:10, Info                  [0x000000] Leaving MigGetRealPlatform method
2024-06-18 08:41:10, Info                  [0x000000] Invalid Platform Architecture
```

**Comment by krisdb2009** [comment]
Can you export the log and upload it here?

**Reply by Deng-Xian-Sheng** [reply]
Thank you
This is the log
Your research in this area should be more extensive than mine, and any suggestions you may provide would be greatly appreciated
[loadstate.log](https://github.com/user-attachments/files/15885984/loadstate.log)

**Reply by Deng-Xian-Sheng** [reply]
> Can you export the log and upload it here?

I suddenly remembered a unique place

I execute scanstatexe on a regular disk in the Win10 system to generate migration storage, and then execute loadstatexe on the Win11 system in vhdx to migrate files to that system

My target system is in vhdx and runs on a physical machine, which is a feature of vhdx that can be booted directly through EFI

I intuitively believe that USMT definitely supports migration between Win10 and Win11, but one is vhdx and the other is not. I don't know if this will affect it

I also have a source system for Win11, which is stored on a regular hard drive. I plan to use it as the source system and the Win11 system in vhdx as the target system, and try migrating again to verify if it is a problem with vhdx or with Win10 and Win11

I have previous experience that this Win11 source system was migrated as a source system to another Win11 system a long time ago, and they were all stored on a regular hard drive and could be migrated normally

If it is indeed due to vhdx, I will try to place the source system in vhdx to generate migration storage, so that both the source and target systems are vhdx

**Reply by Deng-Xian-Sheng** [reply]
It can be confirmed that it is not a problem caused by the migration of Win10 ~ Win11, but rather a problem caused by a regular hard drive ~ vhdx

**Reply by krisdb2009** [reply]
This isn't a SuperGrate log. This page is only for SuperGrate related discussions.

**Reply by Deng-Xian-Sheng** [reply]
But you're just packaging USMT, its logs are still useful, right
Perhaps we should be open and inclusive enough. Those Microsoft guys don't maintain USMT very much, and I haven't found a place to discuss USMT
